src/styles/card.css

```diff
@@ -0,0 +1,111 @@
+.Card {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

Reply:
this comment not fixed.
if you have any questions feel free to ask in chat

Reply:
> this comment not fixed. if you have any questions feel free to ask in chat

Forgot to make a commit after merge, my bad